src/steps/lead-field-equals.ts

```diff
@@ -46,25 +46,24 @@ export class LeadFieldEquals extends BaseStep implements StepInterface {
 
```

Comment:
- [ ] Would also be useful to add the `expectedRecords` property to this step too, including the guaranteed fields recommended for the create step above.